src/spotify_to_tidal/__main__.py

```diff
@@ -24,13 +26,18 @@ def main():
         spotify_playlist = spotify_session.playlist(args.uri)
         tidal_playlists = _sync.get_tidal_playlists_dict(tidal_session)
         tidal_playlist = _sync.pick_tidal_playlist_for_spotify_playlist(spotify_playlist, tidal_playlists)
-        _sync.sync_list(spotify_session, tidal_session, [tidal_playlist], config)
+        _sync.sync_playlists_wrapper(spotify_session, tidal_session, [tidal_playlist], config)
     elif config.get('sync_playlists', None):
         # if the config contains a sync_playlists list of mappings then use that
-        _sync.sync_list(spotify_session, tidal_session, _sync.get_playlists_from_config(spotify_session, tidal_session, config), config)
+        _sync.sync_playlists_wrapper(spotify_session, tidal_session, _sync.get_playlists_from_config(spotify_session, tidal_session, config), config)
+    elif args.sync_favorites:
```

Comment:
I think this `elif` needs to be placed before `elif config.get('sync_playlists', None): `because otherwise the `--sync-favorites` flags gets disregarded as soon as a config file is present.

Reply:
Thanks you're right, though I think if we do this then we also need a config parameter for the default when no command line arg is passed, have added this in a new commit